docs/source/Tutorial.md

```diff
@@ -892,6 +1015,39 @@ a bit more flexibility.
   auto orc = monster_builder.Finish();
 ~~~
 </div>
+<div class="language-c">
+<br>
+*Note: Since we are passing `150` as the `mana` field, which happens to be the
```

Comment:
Please keep the segments specific to C minimal and non-redundant. The issue you are explaining here is not specific to C and is explained elsewhere, so should not be repeated here just for C. All code snippets should be similar to their C++ equivalents, any explanation specific to C would be better off in CUsage.md